---a/net/minecraft/enchantment/EnumEnchantmentType.java
+++b/net/minecraft/enchantment/EnumEnchantmentType.java
@@ -7,8 +7,7 @@
 import net.minecraft.item.ItemSword;
 import net.minecraft.item.ItemTool;
 
-public enum EnumEnchantmentType
-{
+public enum EnumEnchantmentType {
     ALL,
     ARMOR,
     ARMOR_FEET,
@@ -23,34 +23,21 @@
 
     private static final String __OBFID = "CL_00000106";
 
-    public boolean canEnchantItem(Item p_77557_1_)
-    {
-        if (this == ALL)
-        {
+    public boolean canEnchantItem(Item p_77557_1_) {
+        if (this == ALL) {
             return true;
-        }
-        else if (this == BREAKABLE && p_77557_1_.isDamageable())
-        {
+        } else if (this == BREAKABLE && p_77557_1_.isDamageable()) {
             return true;
-        }
-        else if (p_77557_1_ instanceof ItemArmor)
-        {
-            if (this == ARMOR)
-            {
+        } else if (p_77557_1_ instanceof ItemArmor) {
+            if (this == ARMOR) {
                 return true;
-            }
-            else
-            {
-                ItemArmor itemarmor = (ItemArmor)p_77557_1_;
+            } else {
+                ItemArmor itemarmor = (ItemArmor) p_77557_1_;
                 return itemarmor.armorType == 0 ? this == ARMOR_HEAD : (itemarmor.armorType == 2 ? this == ARMOR_LEGS : (itemarmor.armorType == 1 ? this == ARMOR_TORSO : (itemarmor.armorType == 3 ? this == ARMOR_FEET : false)));
             }
-        }
-        else
-        {
+        } else {
             return p_77557_1_ instanceof ItemSword ? this == WEAPON : (p_77557_1_ instanceof ItemTool ? this == DIGGER : (p_77557_1_ instanceof ItemBow ? this == BOW : (p_77557_1_ instanceof ItemFishingRod ? this == FISHING_ROD : false)));
         }
     }
-}
+}
